src/index.js

```diff
@@ -368,7 +368,12 @@ class KadDHT extends EventEmitter {
           })
 
           // run our query
-          timeout((cb) => query.run(rtp, cb), options.timeout)(cb)
+          timeout((_cb) => {
+            query.run(rtp, _cb)
+          }, options.timeout)((res, err) => {
```

Comment:
`res` and `err` order is wrong, they're being passed to the callback in the same order so things are still functioning.

Reply:
Oops, nice catch :)